### DISCUSSION TITLE
Sconti e maggiorazioni su items_list

### DISCUSSION THREAD

**Discussion by donlegro** [fattureincloud/api#107] [Q&A] [answered]
Salve a tutti,
il campo discount accetta solo valori numeri quindi vorrei sapere come gestire multi sconto tipo: 10+5, 5+5+5, ecc.

Inoltre volevo sapere  come gestire uno sconto o maggiorazione (ma soprattutto maggiorazione) con prezzo netto invece di percentuale.
Posso usare ei_raw -> DettaglioLinee -> ScontoMaggiorazione... per quanto riguarda il tracciato xml ma c'è una corrispondenza anche con i campi di Fatture in cloud?

Infine volevo capire l'utilità di discount_highlight che nella prima versione si chiamava sconto_rosso.
Grazie in anticipo per le risposte.

**Comment by emajo** [comment]
Ciao @donlegro, 
1. qual'è il risultato che vuole ottenere per il multi sconto?
2. per gestire sconto/maggiorazione sul totale il campo relativo è `amount_due_discount`, che può essere sia positivo che negativo
3. `discount_highlight` serve per agiungere la dicitura `Sconto a voi riservato: xx% pari a € xx` in fattura

Emanuele

**Reply by donlegro** [reply]
Ciao @emajo e grazie per la risposta,

1. E' il classico multisconto inserito nelle fatture. Cioè se l'importo è 20 euro con sconto 10+5 il totale netto deve essere 20,00-10%= 18,00-5%=17,10 euro che è diverso da 20,00-15%=17,00 euro
2. Ok per sconto incondizionato sul totale documento ma purtroppo chiedevo per sconto/maggiorazione (soprattutto maggiorazione) sul singolo rigo. Faccio un esempio. Alcuni produttori di vino e alcolici aggiungono separatamente una tassa governativa sul rigo che è fissa a prescindere dal numero di bottiglie acquistate invece di spalmarla come fanno altri.
3. Ok tutto chiaro quindi serve solo per attivare la dicitura.

Grazie ancora per l'assistenza.

**Comment by emajo** [comment, marked as answer]
Ciao @donlegro per i punti 1 e 2 purtroppo non disponiamo di campi specifici per quello che vuole fare, quello che consiglio è calcolare gli sconti e maggiorazioni e aggiungerli come fossero degli `items` con nomi descrittivi tipo `sconto generale del xx%` oppure `maggiorazione relativa all'articolo x`.

**Reply by valmoz** [reply]
Ciao @donlegro hai risolto?

Grazie
Mauro

**Reply by donlegro** [reply]
Sì, grazie.
